src/N3Store.js

```diff
@@ -198,11 +198,11 @@ export default class N3Store {
   // ### `_uniqueEntities` returns a function that accepts an entity ID
   // and passes the corresponding entity to callback if it hasn't occurred before.
   _uniqueEntities(callback) {
-    var uniqueIds = Object.create(null), entities = this._entities;
+    var uniqueIds = Object.create(null), entities = this._entities, factory = this._factory;
```

Comment:
I maybe wonder if we want to convert the below to an arrow function instead? Then we don'tt need to store all the variables.

Reply:
I'd be happy to make another PR and change all the nested functions to arrow functions and remove all the `var self = this;` uses and other unnecessary closure creations.

Here is list of some of the areas that would be updated.

- N3Lexer.js in tokenize
- N3Parser.js in parse
- N3Store in _uniqueEntities, import, and remove
- N3StreamParser constructor and import
- N3StreamWriter constructor and import

There may be others too.

Reply:
@dhurlburtusa Thanks, that would be nice indeed! Could you change this single one already for this PR? Then I will merge.

Reply:
Updated.